crates/ast/src/ast/natspec.rs

```diff
@@ -0,0 +1,60 @@
+use super::BoxSlice;
+use crate::token::CommentKind;
+use solar_interface::{Ident, Span, Symbol};
+
+/// A single item within a Natspec comment block.
+#[derive(Clone, Copy, Debug)]
+pub struct NatSpecItem {
+    /// The tag identifier of the item.
+    pub kind: NatSpecKind,
+    /// Span of the tag. '@' is not included.
+    pub span: Span,
+}
+
+/// The kind of a [`NatSpecItem`].
+///
+/// Reference: <https://docs.soliditylang.org/en/latest/natspec-format.html#tags>
+#[derive(Clone, Copy, Debug)]
+pub enum NatSpecKind {
+    Title,
+    Author,
+    Notice,
+    Dev,
+    Param { tag: Ident },
+    Return { tag: Ident },
+    Inheritdoc { tag: Ident },
+    Custom { tag: Ident },
```

Comment:
what do the `tag`s here mean? please also doc comment with the corresponding tag like ``/// `@title` ``